geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/RegionFunctionArgs.java

```diff
@@ -128,6 +132,24 @@ public void setRegionExpirationTTL(Integer timeout, String action) {
     }
   }
 
+  public void setEvictionAttributes(String action, Integer maxMemory, Integer maxEntryCount) {
+    if (action == null) {
+      return;
+    }
+
+    EvictionAction evictionAction = EvictionAction.parseAction(action);
+    if (maxMemory == null && maxEntryCount == null) {
+      evictionAttributes =
+          EvictionAttributes.createLRUHeapAttributes(ObjectSizer.DEFAULT, evictionAction);
```

Comment:
Users are supposed to be able to plugin their own implementation of ObjectSizer. So you should probably support an "eviction-sizer" that takes a class name of the ObjectSizer they want to use for heap or mem lru.